CHANGELOG.md

```diff
@@ -19,13 +19,21 @@ A breaking change will get clearly marked in this log.
   - There's a new field `trade_type` that can be either `orderbook` or `liquidity_pool`. You can filter by that field.
   - Liquidity pool trades will contain the field `liquidity_pool_fee_bp` and either `base_liquidity_pool_id` or `counter_liquidity_pool_id`.
   - There are a few breaking changes to this endpoint listed in the section below.
+- A new kind of trustline called `liquidity_pool_shares` was added and clients can now filter the results from the `/accounts` endpoint by `liquidity_pool`. This will include a new kind of trustline in the account `balances` array, causing a breaking change as described in the section below ([#688](https://github.com/stellar/js-stellar-sdk/pull/688)).
```

Comment:
A bit nit-picky, but maybe this should be two different entries (despite being 1 PR) since they describe different changes:

```suggestion
- A new type of trustline called `liquidity_pool_shares` was added, which is included in the account `balances` array and causes a breaking change (see below) ([#688](https://github.com/stellar/js-stellar-sdk/pull/688)).
- Clients can now filter the results from the `/accounts` endpoint based on participation in a certain liquidity pool ([#688](https://github.com/stellar/js-stellar-sdk/pull/688)).
```

What do you think?

Reply:
Good suggestion, thank you! Addressed in https://github.com/stellar/js-stellar-sdk/pull/688/commits/8ec0cdc93f295652cade3d12f135d4dec0695752.